app/src/ui/history/selected-commit.tsx

```diff
@@ -296,11 +302,20 @@ export class SelectedCommits extends React.Component<
         <div className="panel blankslate">
           <img src={BlankSlateImage} className="blankslate-image" />
           <div>
-            <p>Unable to display diff when multiple commits are selected.</p>
+            <p>
+              Unable to display diff when multiple{' '}
+              {enableMultiCommitDiffs() ? 'non-consecutive ' : ' '}commits are
+              selected.
+            </p>
             <div>You can:</div>
             <ul>
-              <li>Select a single commit to view a diff.</li>
+              <li>
+                Select a single commit{' '}
+                {enableMultiCommitDiffs() ? 'or a range of commits ' : ' '}to
```

Comment:
Should we mention the commits should be consecutive here?
```suggestion
                {enableMultiCommitDiffs() ? 'or a range of consecutive commits ' : ' '}to
```